vrobot_plc_node:
  ros__parameters:
    # Modbus connection parameters
    modbus_tcp_ip: "192.168.10.2"
    modbus_tcp_port: 502
    modbus_tcp_slave_id: 1
    modbus_tcp_timeout: 1000
    modbus_tcp_retry_count: 3
    
    # Retry connection parameters
    max_connection_retries: 5
    retry_delay_ms: 1000
    max_consecutive_failures: 10